src/Projects/Modules/Crunchyroll/Server/Module.Crunchyroll.Cida/Services/CrunchyrollApiService.cs

```diff
@@ -125,27 +128,28 @@ private async Task<string> GetSessionAsync()
 
             return null;
         }
-        
-        
-        public async Task<IEnumerable<Episode>> GetEpisodes(string collectionId)
+
+
+        public async Task<IEnumerable<Episode>> GetEpisodes(string collectionId, CancellationToken cancellationToken = default)
         {
             const string listMediaUrlPart = "list_media.0.json";
 
-                var request = new RestRequest(listMediaUrlPart, Method.GET);
-                request.AddParameter("collection_id", collectionId);
-                request.AddParameter("limit", int.MaxValue);
+            var request = new RestRequest(listMediaUrlPart, Method.GET);
+            request.AddParameter("collection_id", collectionId);
+            request.AddParameter("limit", int.MaxValue);
 
-                return (IEnumerable<Episode>)await this.ExecuteRequestAsync<List<Episode>>(request) ?? Array.Empty<Episode>();
+            return (IEnumerable<Episode>) await this.ExecuteRequestAsync<List<Episode>>(request, cancellationToken) ??
+                   Array.Empty<Episode>();
```

Comment:
```
return (IEnumerable<Episode>) await this.ExecuteRequestAsync<List<Episode>>(request, cancellationToken) ??
                   Enumerable.Empty<Episode>();
```

Reply:
The implementation of `Array.Empty<T>` and `Enumerable.Empty<T>` are both identical. The important thing is, that you should be consistent with it bc both will create an empty array on the first demand and if you mix them you have two of them. Nothing major but yeah. I currently used `Array.Empty<T>` everywhere and `Enumerable.Empty<T>` nowhere so i will stick to the `Array.Empty<T>` to not alter existing code